Configurations/unix-Makefile.tmpl

```diff
@@ -674,6 +674,12 @@ update: generate errors ordinals
 generate: generate_apps generate_crypto_bn generate_crypto_objects \
           generate_crypto_conf generate_crypto_asn1
 
+doc-nits:
+	$(PERL) util/find-doc-nits.pl -n >doc-nits
+	if [ ! -s doc-nits ] ; then exit 0 ; fi
```

Comment:
It was the `exit 0` that worried me.  It will just be the exit code for that line, and `make` will gladly continue with the rest of the lines...